### PR TITLE
Adding the flipping animation in the two axes (X and Y)

### PR DESCRIPTION
**Pull request by MalekMK**
This PR is related to the issue #1 , it provides missing Flip animations: 
- flip
- flipInX
- flipOutX
- flipInY
- flipOutY

### CODE REVIEW

File: dist/vue2-animate.css [hunk 1]
```diff
@@ -1752,4 +1752,332 @@
...
+.flip-enter-active, .flipIn {
+  -webkit-animation-name: flipIn;
+          animation-name: flipIn; }
+
+.flip-leave-active, .flipOut {
+  -webkit-animation-name: flipOut;
+          animation-name: flipOut; }
+
+.flipflip-enter-active, .flipInflip,
+.flipflip-leave-active, .flipOutflip {
```
Comment (asika32764) [inline dist/vue2-animate.css]: This name is duplicated,

**Comment by asika32764** [comment]
Thank you for your work. 

Adding `flip` or `flipIn` on HTML class is work. But not work for Vue transition since there are some wrong class names.

**Comment by MalekMK** [comment]
Thank you for your feedback. I've fixed the bugs , you can check the commits .

**Comment by asika32764** [comment]
Thank you, please wait me test

**Pull request merged**